lib/movie_overview.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'movie.dart';
+import 'plain_text.dart';
+import 'ui_constants.dart';
+
+class MovieOverview extends StatelessWidget {
+  const MovieOverview({
+    super.key,
+    required this.movie,
+  });
+
+  final Movie movie;
+  static const String overview = 'Overview';
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+      const Text(
+        overview,
+        style: UIConstants.subtitleTextStyle,
+      ),
+      UIConstants.lineSpace,
+      PlainText(
+        text: movie.overview,
+      ),
+    ]);
```

Comment:
trailing comma, and use the format shortcut to fix the indentation

Reply:
done

Reply:
In fact, run the auto-format for the whole project to fix the `import`s order as well. You can click the `lib` folder in the IDE and then use the shortcut.